sources/packages/forms/src/form-definitions/reportscholasticstandingchange.json

```diff
@@ -2543,12 +2450,12 @@
                                             "value": ""
                                         }
                                     ],
-                                    "content": "Final Study Costs",
+                                    "content": "${{ data.applicationOfferingProgramRelatedCosts ?? 0}}",
```

Comment:
The money values along the form do not seem to be formatted. We need to ensure the format defined on Figma as $10000.00 (using .toFixed(2)).